### PR TITLE
CashRegister Class created and methods added

### PR DESCRIPTION
**Pull request by mgm702**
Created the CashRegister Class and made sure that all of the methods work in irb 

- Made sure that the class had no Rubocop offenses. 
- Made sure that the output statements are accurately reduced to the two decimal places.
- Used attr_reader instead of total method to return the total instance variable.

[#105433772]

### CODE REVIEW

File: objects/cash_register.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Here is the CashRegister class
+class CashRegister
+  attr_reader :total
+  def initialize
+    @total = 0
```
Comment (jasonnoble) [inline objects/cash_register.rb]: If you use `@total = 0.0` here, all the math below will be using floating point math.

File: objects/cash_register.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  end
+
+  def purchase(user_value)
+    @total += user_value.to_f
+    @total
```
Comment (jasonnoble) [inline objects/cash_register.rb]: No need for this line.  The line above it will return the same thing.

File: objects/cash_register.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Here is the CashRegister class
+class CashRegister
+  attr_reader :total
+  def initialize
+    @total = 0
+  end
+
+  def purchase(user_value)
```
Comment (jasonnoble) [inline objects/cash_register.rb]: `amount`?

File: objects/cash_register.rb [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    @total += user_value.to_f
+    @total
+  end
+
+  def pay(user_value)
+    @total -= user_value.to_f
+    if @total < 0
```
Comment (jasonnoble) [inline objects/cash_register.rb]: What if it's equal to 0?

**Comment by mgm702** [comment]
I made the corrections you suggested to my CashRegister class. It runs the same, but the code looks much cleaner and it has less lines which is always a plus.

**Pull request merged**